plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraServer.java

```diff
@@ -67,17 +69,25 @@
     public CassandraServer()
             throws Exception
     {
-        this("2.2");
+        this("cassandra:2.2", ImmutableMap.of());
     }
 
-    public CassandraServer(String cassandraVersion)
+    public CassandraServer(String imageName)
+            throws Exception
+    {
+        this(imageName, ImmutableMap.of());
+    }
+
+    public CassandraServer(String imageName, Map<String, String> additionalEnvVariables)
```

Comment:
We avoid abbreviations as much as possible. Let's rename to `environmentVariables`.